packages/web/src/components/audio-transactions-table/AudioTransactionsTable.tsx

```diff
@@ -93,7 +93,11 @@ const renderTransactionTypeCell = (cellInfo: TransactionCell) => {
 
 const renderBalanceCell = (cellInfo: TransactionCell) => {
   const transaction = cellInfo.row.original
-  return transaction.balance ? formatAudio(transaction.balance) : ''
+  return transaction.balance
+    ? wAUDIO(BigInt(transaction.balance)).toLocaleString('en-US', {
```

Comment:
super duper nit but `en-US` doesn't make tons of sense outside of doing USD <=> USDC, maybe leave undefined or null or whatever